chain/indexer/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 
 use tokio::sync::mpsc;
 
+use anyhow::Result;
```

Comment:
Following the existing style, this use statement should belong to the above block.

Reply:
Thanks for noticing, fixed.